packages/patterns-react/src/internal/FeatureFlags.js

```diff
@@ -77,3 +77,14 @@ export const DDS_CARDS_WITHOUT_IMAGES =
  */
 export const DDS_USECASES =
   process.env.DDS_USECASES === 'true' || DDS_FLAGS_ALL || false;
+
+/**
+ * This determines if the cards with images will be rendered or not
+ *
+ * @type {string | boolean}
+ */
+export const DDS_CARDS_WITH_IMAGES =
+  process.env.DDS_CARDS_WITH_IMAGES === 'true' || DDS_FLAGS_ALL || false;
```

Comment:
Is `DDS_CARDS_WITH_IMAGES` needed anymore if we are consolidating to `DDS_CARDS_SECTION`?

Reply:
yeah @jeffchew, we do not need, removed and updated.